core/src/test/java/com/google/cloud/sql/core/CloudSqlInstanceTest.java

```diff
@@ -138,13 +138,16 @@ public void testCloudSqlInstanceForcesRefresh() throws Exception {
             stubCredentialFactory,
             executorService,
             keyPairFuture,
-            RateLimiter.create(1.0 / 30.0));
+            TEST_RATE_LIMITER);
+
+    instance.getSslData();
+    assertThat(refreshCount.get()).isEqualTo(1);
 
-    SslData gotSslData = instance.getSslData();
-    assertThat(gotSslData).isSameInstanceAs(sslData);
     instance.forceRefresh();
+
     instance.getSslData();
-    assertThat(refreshCount.get()).isEqualTo(2);
+    // refresh count hasn't changed because we re-use the existing connection info
+    assertThat(refreshCount.get()).isEqualTo(1);
```

Comment:
We need to add on to this test that forceRefresh() still works. That eventually `assertThat(refreshCount.get()).isEqualTo(2);` is true.